src/common/framework/util/util.ts

```diff
@@ -1,5 +1,12 @@
 import { timeout } from './timeout.js';
 
+export class StacklessError extends Error {
```

Comment:
nit: Comment why we have this. Seems like a convenient way to do logging but also exit quickly from a function.